tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -13,6 +13,31 @@ module.exports = {
       return navigationHelper.navigateAndWaitTillLoaded(
         this.url(), this.page.FilesPageElement.filesList().elements.filesListProgressBar.selector
       )
+    },
+    declinedStatusForDeletedFile: function (filename) {
+      return this.waitForElementVisible({
```

Comment:
I'd suggest we check the text of the elements for the declined status. This way we can have better error handling/logging instead of `"not found"` in the log.